framework/include/kernels/Kernel.h

```diff
@@ -70,8 +70,14 @@ class Kernel : public KernelBase, public MooseVariableInterface<Real>
   /// Time derivative of u
   const VariableValue & _u_dot;
 
+  /// Second time derivative of u
```

Comment:
I remember @friedmud and @bwspenc told me that their master plan is to move these time derivatives from Kernel to TimeKernel so that it's a bit more organized and easier to manage.

Reply:
For the dynamics kernels in tensor mechanics to work with ActuallyExplicitEuler (or the central difference version of it), it looks like the dynamics kernels have to derive from TimeKernel instead of Kernel. If we make that change, then the first and second time derivatives of the solution can be moved to TimeKernel from Kernel.

@friedmud and @bwspenc, shall I go ahead and make this change?